appium/webdriver/webelement.py

```diff
@@ -58,22 +56,19 @@ def get_attribute(self, name):
         if attributeValue is None:
             return None
 
-        if not isinstance(attributeValue, str):
```

Comment:
I would say it is still needed

Reply:
Here is only for python2 since `unicode` is only in python2, so it's unnecessary.

Reply:
not really. This is to make the spec happy, which says that attribute value could either be a string or null, but not boolean, number or something else

Reply:
Ah, thanks, I haven't understood codes' meaning.
I restored and modified to `str` from `unicode`.

Reply:
Please add an explanation comment to this block of the code, so it would be more clear to other people, who'd read it